tox.ini

```
@@ -102,6 +102,7 @@ deps =
     requests
     types-PyYAML
     types-requests
+    websockets<14.0 # https://github.com/juju/python-libjuju/issues/1184
```

Would it be possible to briefly summarize the issue here please?
```suggestion
    ; 2024/11/19 - there is an incompatibility issue with latest websockets lib release and pylib juju
    websockets<14.0 # https://github.com/juju/python-libjuju/issues/1184
```